apps/api/src/planning-session/planning-session.service.ts

```diff
@@ -104,6 +109,15 @@ export class PlanningSessionService {
     return;
   }
 
+  async exportCsv(sessionId: string): Promise<any> {
+    try {
```

Comment:
Apologies, this try..catch will not be needed since we already have global exception filter in error-exception.filter.ts